imports/ui/components/user_mgmt/user_mgmt.js

```diff
@@ -4,10 +4,12 @@ import { Meteor } from 'meteor/meteor';
 import { EasySearch } from 'meteor/easy:search';
 import '../../../api/users/users_collection.js';
 import { UsersIndex } from '../../../api/users/users_index.js';
+import { Profiles } from '../../../api/users/profiles_collection.js';
 
 Template.userMgmt.onCreated(function() {
   var self = this;
-  self.subscribe("users");
+  self.subscribe('users');
+  self.subscribe('profiles');
```

Comment:
These subscriptions could be a heavy load on the server if the user base grows large, but I'm not sure if you want to address this now. Would paginate solve this issue? Anyway, probably not a big worry as the ban system is only going to be used sporadically and our number of users will probably not be very large.